schema.sql

```diff
@@ -50,3 +50,10 @@ CREATE TABLE visits (
   vet_id INTEGER REFERENCES vets(id),
   visit_date DATE
 );
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+ CREATE INDEX owners_idx ON owners (id);
+ CREATE INDEX visits_vet_id_idx ON visits (vet_id);
+ 
```

Comment:
Great job team🤝

- [ ] As I can see from the screenshots provided in the PR, only one query `SELECT * FROM visits where vet_id = 2`; used index to improve its performance. The evidence is the index scan in the output.
![=2](https://user-images.githubusercontent.com/63917725/232590456-999e0c47-9067-42cc-8d18-2bd01e413d06.PNG)

- [ ] You can use indexes to improve performance for the other two queries (`SELECT COUNT(\*) FROM visits where animal_id = 4`; and `SELECT \* FROM owners where email = 'owner_18327@mail.com'`;). Kindly make sure there is an index scan in the second output the after the improvements with adding indexes. There is no index scan in the outputs of these two commands.
![=4](https://user-images.githubusercontent.com/63917725/232591575-e959c439-44b4-4299-838d-4d0736a3aeec.PNG)